.jenkins/actions/run_standalone.sh

```diff
@@ -45,17 +46,25 @@ TIMESTEPS=60
 RANKS=6
 BENCHMARK_DIR=${ROOT_DIR}/examples/standalone/benchmarks
 DATA_DIR="/project/s1053/fv3core_serialized_test_data/${DATA_VERSION}/${experiment}"
-ARTIFACT_DIR="/project/s1053/performance/fv3core_monitor/${backend}"
+ARTIFACT_ROOT="/project/s1053/performance/"
+TIMING_DIR="${ARTIFACT_ROOT}/fv3core_monitor/${backend}"
+PROFILE_DIR="${ARTIFACT_ROOT}/fv3core_profile"
```

Comment:
You probably want `${backend}` here too

Reply:
Potentially we also want to think about a full UUID for those, which in our case would be (experiments, backend).

Reply:
> You probably want ${backend} here too
done

Reply:
> Potentially we also want to think about a full UUID for those, which in our case would be (experiments, backend).

keeping a history of those can be quite expensive. I think i would keep with the current logic of just having latest until we see a need for it, ok?